src/Servers/HttpSys/src/HttpSysOptions.cs

```diff
@@ -23,11 +24,30 @@ public class HttpSysOptions
         private RequestQueue _requestQueue;
         private UrlGroup _urlGroup;
         private long? _maxRequestBodySize = DefaultMaxRequestBodySize;
+        private string _requestQueueName;
```

Comment:
Where is this value actually used? I feel like this change originally passed the request queue name somewhere but now it doesn't.

Reply:
Yeah, I think you lost something in the rebase.